app/src/actions/user.js

```diff
@@ -57,7 +57,7 @@ export function logout() {
       type: LOGOUT
     });
     window.location.hash = window.location.hash.replace(/#access_token=([a-zA-Z0-9.\-_]*)/g, '');
-    if (window.location.pathname.match('^\/map')) {
+    if (window.location.pathname.match('/map')) {
```

Comment:
@hectoruch what was the error? <img src="http://1.bp.blogspot.com/-gTloqUf-lgQ/VBopRP9dXXI/AAAAAAAAH-U/SKcaqt-z-QU/s1600/tumblr_inline_nbtekxdC1x1qeisx5.gif" />